pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -215,6 +251,17 @@ func (r *{{ .Resource.Kind }}Reconciler) Reconcile(ctx context.Context, req ctrl
 		if err = r.Update(ctx, found); err != nil {
 			log.Error(err, "Failed to update Deployment", 
 				"Deployment.Namespace", found.Namespace, "Deployment.Name", found.Name)
+
+			// The following implementation will update the status
+			meta.SetStatusCondition(&{{ lower .Resource.Kind }}.Status.Conditions, metav1.Condition{Type: typeDegraded{{ .Resource.Kind }},
+				Status: metav1.ConditionTrue, Reason: "Resizing",
```

Comment:
```suggestion
				Status: metav1.ConditionFalse, Reason: "Resizing",
```

We could not  resising